src/app.js

```diff
@@ -1,3 +1,4 @@
+require('dotenv').config();
 const express = require('express');
 const app = express();
 const cors = require('cors')
```

Comment:
[Functional Error]
You haven't called the `connectDB` function in your app.js so its not actually connecting to the db.
Check your example app.js again to see how that's done. The code looks right, its just not actually getting called.

Reply:
called connectDB in server.js, so now it's connecting to the db